src/modules/Game.class.js

```diff
@@ -21,24 +21,176 @@ class Game {
    * initial state.
    */
   constructor(initialState) {
-    // eslint-disable-next-line no-console
-    console.log(initialState);
+    this.score = 0;
+    this.status = 'idle'; // One of: 'idle', 'playing', 'win', 'lose'
```

Comment:
That is why you should not use JS for a modern real world tasks
status should be an enum (TS has support for enums and for union types)

Still, this is error prone to have a status like this - with simple literals

Step 1: add following code 

```
const Status = {
Idle: 'idle',
Playing: 'playing',
// ...
}
```

Step 2 

```suggestion
    this.status = Status.Idle;
```

See more on enums in JS: https://www.sohamkamani.com/javascript/enums/

Reply:
Step 3 - never compare the status with a literal directly - always use your object

`if (this.status === Status.Playing) ... `